### DISCUSSION TITLE
can it work with TrueNas scale

### DISCUSSION THREAD

**Discussion by gaodaxiaya** [blakeblackshear/frigate#2070] [Ask A Question]
can it work with TrueNas scale？

**Comment by blakeblackshear** [comment]
I have never used it before, but it appears to be debian based and can run containers. It can likely run Frigate, the question is whether or not the Coral device is compatible and can be passed to containers. I would guess that the USB Coral can at least.

**Reply by changwah** [reply]
Just wondering if anyone has gotten it working in TrueNas Scale. I am thinking of migrating to Scale and I have the M.2 Tensor unit.
